### ISSUE TITLE
Does not work with Typescript

### ISSUE THREAD

**Issue by adamduren** [getsentry/sentry-webpack-plugin#27]
When attempting use use with Webpack I get the following:

```
Error: ./src/app/app.module.ts
Module parse failed: Unexpected character '@' (27:0)
You may need an appropriate loader to handle this file type.
| import { Services } from '../services';
|
| @NgModule({
|   declarations: [MyApp],
|   imports: [
 @ ./src/app/main.ts 16:0-41
 @ multi ./node_modules/@sentry/webpack-plugin/src/sentry-webpack.module.js ./src/app/main.ts
    at new BuildError (/path-to-project/node_modules/@ionic/app-scripts/dist/util/errors.js:16:28)
    at callback (/path-to-project/node_modules/@ionic/app-scripts/dist/webpack.js:121:28)
    at emitRecords.err (/path-to-project/node_modules/webpack/lib/Compiler.js:269:13)
    at Compiler.emitRecords (/path-to-project/node_modules/webpack/lib/Compiler.js:375:38)
    at emitAssets.err (/path-to-project/node_modules/webpack/lib/Compiler.js:262:10)
    at applyPluginsAsyncSeries1.err (/path-to-project/node_modules/webpack/lib/Compiler.js:368:12)
    at next (/path-to-project/node_modules/tapable/lib/Tapable.js:218:11)
    at Compiler.compiler.plugin (/path-to-project/node_modules/webpack/lib/performance/SizeLimitsPlugin.js:99:4)
    at next (/path-to-project/node_modules/tapable/lib/Tapable.js:220:14)
    at versionPromise.then.then.then.then (/path-to-project/node_modules/@sentry/webpack-plugin/src/index.js:92:21)
```

**Comment by adamduren** [comment]
It appears I was able to fix this by modifying the `injectRelease` function as such to convert from loaders syntax to rules if loaders is used. The old loaders syntax is used in Ionic. This change is probably not comprehensive but seems to get the job done.

```javascript
  // added code to convert from loaders syntax to rules syntax
  if (typeof changedCompiler.options.module.loaders !== 'undefined') {
    changedCompiler.options.module.loaders.map(loader => {
      changedCompiler.options.module.rules.push({
        test: loader.test,
        use: [{
          loader: loader.loader,
        }],
      });
    });
  }

  delete changedCompiler.options.module.loaders;
```

**Comment by adamduren** [comment]
Upon thinking about it further I believe it would be better to warn the user about the incompatibility vs attempting to rewrite their webpack config. Thoughts?

**Comment by kamilogorek** [comment]
Fix published as `1.3.3`. Please let us know if it solves the issue :)
